### ISSUE TITLE
Remove quotes in mappyfile.findall() 

### ISSUE THREAD

**Issue by kdeininger** [geographika/mappyfile#61]
The function `mappyfile.findall()` takes the specified value and adds single and double quotes as possible values to search for:

https://github.com/geographika/mappyfile/blob/d253620f3fe6e686b99d796ce86b71d5c8b2eed5/mappyfile/utils.py#L239-L240

But the result of the parsed mapfile contains the values (e.g. group or layer names) without any quotes and `mappyfile.findall()` always returns an empty generator object.

In my case, I could fix it by passing the specified value directly:

```python
return (item for item in lst if item[key.lower()] == value)
```

Is there something wrong with my parsed content or needs this function to be fixed? `mappyfile.find()` works fine.

**Comment by geographika** [comment]
@kdeininger - thanks for reporting this. The addition of quotes are left over from early versions of mappyfile which didn't strip them automatically from the parse tree. 
I'll add the fix and a test for this shortly.

**Comment by geographika** [comment]
Fixed in 0.7.2 release
